### PR TITLE
Removed duplicated array keys in Inflector.

### PR DESCRIPTION
**Pull request by nahakiole**
I've found a few duplicated array keys in the Inflector class. This pull request would remove the duplicated keys and values.

The value of the duplicates is the same so this won't affect any functionality.

### CODE REVIEW

File: src/Utility/Inflector.php [hunk 1]
```diff
@@ -304,7 +301,6 @@ class Inflector
         'Ơ' => 'O',
         'Ø' => 'O',
         'Ǿ' => 'O',
-        'Ö' => 'Oe',
```
Comment (markstory) [inline src/Utility/Inflector.php]: I don't see a duplicate of ths.

Comment (nahakiole) [inline src/Utility/Inflector.php]: The duplicate is here: https://github.com/cakephp/cakephp/pull/7558/files#diff-19eff47ebbcb312510b116f11192a975R176

File: src/Utility/Inflector.php [hunk 2]
```diff
@@ -358,7 +354,6 @@ class Inflector
         'Ǘ' => 'U',
         'Ǚ' => 'U',
         'Ǜ' => 'U',
-        'Ü' => 'Ue',
```
Comment (markstory) [inline src/Utility/Inflector.php]: Also not a dupe. This is u01db and the 'duplicate' is u00dc

File: src/Utility/Inflector.php (hunk 1 above)
Comment (markstory) [inline src/Utility/Inflector.php]: I missed that - sorry.

File: src/Utility/Inflector.php (hunk 2 above)
Comment (markstory) [inline src/Utility/Inflector.php]: The dupe is on line 175 - my bad.

**Pull request merged**